services/system/Accounts/ui/src/AccountSelection.tsx

```diff
@@ -136,20 +154,21 @@ export const AccountSelection = () => {
 
     const [activeSearch, setActiveSearch] = useState("");
     const accountsToRender = activeSearch
-        ? (accounts || []).filter((account) =>
+        ? (accounts || []).filter((account: { account: string }) =>
               account.account
                   .toLowerCase()
                   .includes(activeSearch.toLowerCase()),
           )
         : accounts || [];
-    // const [accountsToRender, setAccountsToRender]: [
-    //   AccountType[],
-    //   (accounts: AccountType[]) => void,
-    // ] = useState<AccountType[]>([]);
 
-    // useEffect(() => {
-    //   setAccountsToRender(accounts || []);
-    // }, [accounts]);
+    useEffect(() => {
+        async function preloadAuthAny() {
+            await supervisor.preLoadPlugins([
+                { service: "auth-any", plugin: "plugin" },
+            ]);
+        }
+        preloadAuthAny();
+    }, []);
```

Comment:
@James-Mart meant to ask you about this. I get a failure if I don't preload this plugin. Is this expected? or a bug (known or not?)?

Reply:
What failure are you seeing? If I comment out your code to do this preload here, I don't get any errors in the console

Reply:
<img width="782" height="433" alt="image" src="https://github.com/user-attachments/assets/dc60d9cf-aeda-4d6b-8310-b5ff23701e17" />